### DISCUSSION TITLE
NuGet.Frameworks latest version doesn't support NET8

### DISCUSSION THREAD

**Discussion by Paciv** [NuGet/Home#13082] [Q&A] [answered]
Hello,

Support for NET8 in NuGet.Frameworks package has been merged since 10/3: https://github.com/NuGet/Home/issues/12910
And NET8 went out on 11/14, but the 6.8 release (11/17) does not contains the change

When is it expected to have NET8 support in NuGet.Frameworks released ?

**Comment by JonDouglas** [comment, marked as answer]
@Paciv Likely the following release. To be clear, .NET 8 is supported just that the constants are missing.

**Reply by Paciv** [reply]
Yes, I had a doubt when looking only at the switch part of the change.
I checked the code further and the parsing still works without it.

Still, it would have been nice to have the constants now that NET8 is out.
